docker/docker-compose.yml

```diff
@@ -84,7 +84,7 @@ services:
     volumes:
       - addons:/tmp/addons:ro # readonly
     environment:
-      - WORKER_WATCHING_QUEUES=training,pipelines
+      - WORKER_WATCHING_QUEUES=training
```

Comment:
this makes local development easier.  when I'm working on pipeline code, I need to know which celery worker to launch in dev mode instead of having to launch both.